src/components/Search.js

```diff
@@ -1,41 +1,115 @@
-import React from 'react';
-
-const Search = props => (
-  <div className="search">
-    <div className="page-header">
-      <h4 className="text-left">Search Bookings</h4>
-    </div>
-    <div className="row">
-      <div className="col">
-        <table className="table search-table">
-          <thead>
-            <tr>
-              <th>Customer id</th>
-              <th />
-              <th></th>
-              <th />
-            </tr>
-          </thead>
-          <tbody>
-            <tr>
-              <td>
-                <input
-                  id="customerId"
-                  type="text"
-                  className="form-control"
-                  placeholder="Customer id"
-                  name="customerId"
-                />
-              </td>
-              <td>
-                <button className="btn btn-primary fn-submit-name">Search IDs</button>
-              </td>
-              {/* Add search by name here */}
-            </tr>
-          </tbody>
-        </table>
+import React, { Component } from 'react';  
+
+class SearchInput extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render () {
+    return (
+      <input
+        id={this.props.inputId}
+        type="text"
+        className="form-control"
+        placeholder={this.props.placeholder}
+        name={this.props.name}
+        onChange={this.props.handleInput}
+      />
+    );
+  }
+}
+
+class SearchButton extends Component {
```

Comment:
Using multiple components in a file can be a nice way to condense the number of files, but in a bigger project I'd say one file per component is probably best